std/typecons.d

```diff
@@ -7143,3 +7143,150 @@ unittest
         ubyte, ubyte, T3, T3,
     );
 }
+
+/**
+Ternary type with three thruth values.
+*/
```

Comment:
I know it's redundant, but it probably doesn't hurt to state those three values here